src/Symfony/Component/Translation/Exception/ProviderException.php

```diff
@@ -20,8 +20,8 @@
  */
 class ProviderException extends RuntimeException implements ProviderExceptionInterface
 {
-    private $response;
-    private $debug;
+    private ResponseInterface $response;
+    private string $debug;
```

Comment:
this looks fragile: https://3v4l.org/Epn4h (not sure the concat assignment is intended somehow :/)

Reply:
Good catch. I'd remove the concatenation, tbh. Looks like a mistake.

Reply:
im curious why psalm doesnt notice :/ https://psalm.dev/r/1f32ea070a

Reply:
Given the `\Symfony\Contracts\HttpClient\ResponseInterface::getInfo` docblock can even return an array,  I've written a test to verify that it works in e01134c41840e69045be39f700c711fbd4c76415

Yes, I also think we can see it as typo. [This commit](https://github.com/symfony/symfony/commit/6e55fa84b7ed9751784b4c478d8e16b22062ed93) on 5.3 adds that provider as experimental, but `ProviderException` [is unknown at the `php-translation` organization](https://github.com/search?q=org%3Aphp-translation+ProviderException&type=source) so does not have any other history.

Reply:
> im curious why psalm doesnt notice :

Probably because the related error gets included with the baseline errors, as that concat already was part of the codebase before.

> ERROR: UninitializedProperty - 6:9 - Cannot use uninitialized property $this->s